package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "movies-explorer-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node app.js",
+    "dev": "nodemon app.js",
+    "lint": "npx eslint .",
+    "lint:fix": "npx eslint . --fix"
+  },
+  "keywords": [],
+  "author": "Andrei Maksimenya",
+  "license": "ISC",
+  "devDependencies": {
+    "eslint": "^8.44.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.27.5",
+    "nodemon": "^2.0.22"
+  },
+  "dependencies": {
+    "bcrypt": "^5.1.0",
```

Comment:
Кстати, в лекциях у вас был пакет `bcryptjs`. Не понимаю, почему народ переодически этот ставит вместо него.

Reply:
Некоторые может просто по невнимательности, там действительно 2 пакета схожих. Но я читал [сравнения](https://codeforgeek.com/bcrypt-vs-bcryptjs/),  там в конце написано :

> When it comes to the backend or server-side NodeJS Applications it is wiser to choose the Native Bcrypt library because of the significant performance advantages over BcryptJS. 

Но был и [stackoverflow](https://stackoverflow.com/questions/54674387/how-do-nodes-bcrypt-and-bcryptjs-libraries-differ) указывающий обратно. Так что, до конца не понял. Один на С, другой на JS.